pkg/odo/cli/version/version.go

```diff
@@ -2,11 +2,10 @@ package version
 
 import (
 	"fmt"
-	"os"
-	"strings"
-
 	"github.com/openshift/odo/pkg/occlient"
 	"github.com/openshift/odo/pkg/odo/genericclioptions"
+	"os"
+	"strings"
```

Comment:
no changes here.. I believe your `goimports` too may of messed up :) just move it back to the top!